src/Components/Profile/MyPosts/Post/post.jsx

```diff
@@ -5,16 +5,23 @@ import style from './post.module.css';
 import logo from './post.jpg';
 
 const Post = (props) => {
-  const { message } = props;
-  return (
-    <div className={style.item}>
+  const { profileData } = props.messages;
+  const PostData = Object.values(profileData);
```

Comment:
The naming of this variable is confusing. The fact that it starts with capital letter implies that it's either a Class, a Constructor function or a Component.
But in reality it's just some data - so consider following the naming conventions and start the names for such variables with lowercase letter.
https://github.com/airbnb/javascript#naming-conventions

Reply:
I've read this text.It was very useful for me